net/MapViewController.m

```diff
@@ -126,6 +129,23 @@ -(void)viewWillAppear:(BOOL)animated
     [runner addTimer:self.timerForMapRenew forMode: NSDefaultRunLoopMode];
 }
 
+-(void)selectCurrentMarker
+{
+        if(self.isMarkerSelected == YES) {
```

Comment:
it's enough to have mapView.selectedMarker
The condition will be if (self.mapView.selectedMarker != nil)
Is there any reason to have isMarkerSelected property?